### PR TITLE
Remove RollbarPLCrashReporter library.

### PR DESCRIPTION
**Pull request by matux**
## Description of the change

Taking a few steps to simplify how we collect crashes. I'm dropping this library in favor of another that does a better job of capturing OS signals.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Maintenance
- [ ] New release

## Related issues

- Fix [119946](https://app.shortcut.com/rollbar/story/119946/apple-sdk-retrieve-stack-traces-for-all-types-of-crashes)

### Development

- [x] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [x] All tests related to the changed code pass in development

### Code review

- [x] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [x] "Ready for review" label attached to the PR and reviewers assigned
- [x] Issue from task tracker has a link to this pull request
- [ ] Changes have been reviewed by at least one other engineer

### CODE REVIEW

File: Demos/iosAppSwift/iosAppSwift/iosAppSwiftApp.swift [hunk 1]
```diff
@@ -22,7 +22,7 @@ class AppDelegate: NSObject, UIApplicationDelegate {
     ) -> Bool {
         // Dynamically read these settings from your config settings on
         // application startup.
-        let accessToken = "YOUR-ROLLBAR-ACCESSTOKEN"  // Rollbar post_client_item access token
+        let accessToken = "2fa95b91f38448299b5097ff11cffb23"  // Rollbar post_client_item access token
```
Comment (cyrusradfar) [inline Demos/iosAppSwift/iosAppSwift/iosAppSwiftApp.swift:25]: @matux we need to revoke this token and remove it, right?

**Review by cyrusradfar — CHANGES_REQUESTED** [review]
We should remove the ticket and then resubmit.

File: Demos/iosAppSwift/iosAppSwift/iosAppSwiftApp.swift (hunk 1 above)
Comment (matux) [reply Demos/iosAppSwift/iosAppSwift/iosAppSwiftApp.swift]: Not sure about revoking, I'll check the account I'm using. I just removed it from the code, tho.

**Review by cyrusradfar — APPROVED** [review]
Thank you, I appreciate how you improved that clarity of the code and shortened it. Quite an accomplishment.

**Pull request merged**